planner/core/testdata/plan_suite_out.json

```diff
@@ -1683,14 +1683,14 @@
       {
         "SQL": "select /*+ HASH_AGG(), AGG_TO_COP() */ sum(distinct b) from pt;",
         "Plan": [
-          "HashAgg_11 1.00 root  funcs:sum(distinct Column#7)->Column#4",
-          "└─Projection_23 16000.00 root  cast(test.pt.b, decimal(65,0) BINARY)->Column#7",
+          "HashAgg_11 1.00 root  funcs:sum(distinct Column#9)->Column#4",
+          "└─Projection_23 16000.00 root  cast(test.pt.b, decimal(65,0) BINARY)->Column#9",
           "  └─Union_12 16000.00 root  ",
-          "    ├─HashAgg_16 8000.00 root  group by:test.pt.b, funcs:firstrow(test.pt.b)->test.pt.b",
+          "    ├─HashAgg_16 8000.00 root  group by:test.pt.b, funcs:firstrow(test.pt.b)->test.pt.b, funcs:firstrow(test.pt.b)->test.pt.b",
```

Comment:
The `firstrow` is duplicated here?

Reply:
Yes. For correctness.

Reply:
Can we check if the firstrow is already existed? 
BTW, I'm confused why we need to add firstrow here, it is supposed be added by planbuilder？

Reply:
Because the partial aggregation should return the group by columns. In that way, final aggregation could execute.